Client/Impl/Worker/JobWorker.cs

```diff
@@ -77,10 +86,95 @@ internal void Open()
         {
             isRunning = true;
             var cancellationToken = source.Token;
-            var jobWorkerSignal = new JobWorkerSignal();
 
-            StartPollerThread(jobWorkerSignal, cancellationToken);
-            StartHandlerThreads(jobWorkerSignal, cancellationToken);
+            activateJobsCommand = jobWorkerBuilder.Command;
+            maxJobsActive = jobWorkerBuilder.Command.Request.MaxJobsToActivate;
+
+            // Create options for blocks
+            var bufferOptions = new DataflowBlockOptions
+            {
+                CancellationToken = cancellationToken,
+                EnsureOrdered = false
```

Comment:
We could also limit the buffer to maxJobsActive

Reply:
You could do that - but it does not help. Limiting the buffer means Post(Async) may fail, but you already activated the job - so it does not help.